build.gradle

```diff
@@ -0,0 +1,31 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ *
+ * This generated file contains a sample Java Library project to get you started.
+ * For more details take a look at the Java Libraries chapter in the Gradle
+ * User Manual available at https://docs.gradle.org/5.5.1-20190724234647+0000/userguide/java_library_plugin.html
+ */
+
+plugins {
+    // Apply the java-library plugin to add support for Java Library
+    id 'java-library'
+}
+// Set library info
+version = "0.0.1"
+group = "com.frc4905"
```

Comment:
I kinda just picked a [versioning system](https://semver.org) and group (based on the website)